.github/workflows/pr.yml

```diff
@@ -0,0 +1,25 @@
+name: pr
```

Comment:
repos generally break these up by workflow (i.e. run lint in a separate workflow from test and whatever else). this workflow name makes me think you're going to throw all of the steps into this file as things become more complex. is that correct? if not, i recommend renaming to `lint`

Reply:
I'm going to add testing in the next PR. What about the name "list-test"?